interactions/api/models/message.py

```diff
@@ -790,6 +790,19 @@ async def delete(
             guild_id=guild_id, emoji_id=int(self.id), reason=reason
         )
 
+    @property
+    def url(self) -> str:
+        """
+        Returns the emoji's url.
+        :return URL of the emoji
+        :rtype: str
+        """
+        if not self._client:
+            raise AttributeError("HTTPClient not found!")
```

Comment:
This doesn't need the client attribute to function

Reply:
OK, I followed and removed the ``_client`` function.